t/plugin/request-validation.t

```diff
@@ -1449,3 +1448,147 @@ passed
 200
 --- no_error_log
 [error]
+
+
+
+=== TEST 35: add route (test request validation `header_schema.required` success with custom reject message)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                    "plugins": {
+                        "request-validation": {
+                            "header_schema": {
+                                "type": "object",
+                                "properties": {
+                                    "test": {
+                                        "type": "string",
+                                        "enum": ["a", "b", "c"]
+                                    }
+                                },
+                                "required": ["test"]
+                            },
+                            "rejected_message": "customize reject message"
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1982": 1
+                        },
+                        "type": "roundrobin"
+                    },
+                    "uri": "/plugin/request/validation"
+                }]])
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- request
+GET /t
+--- response_body
+passed
+--- error_code chomp
+200
+--- no_error_log
+[error]
+
+
+
+=== TEST 36: add route (test request validation `body_schema.required` success with custom reject message)
```

Comment:
TEST 35 & 36 are the same?

Reply:
No, TEST 35 test for `header_schema`, TEST 36 test for `body_schema`, with same properties

Reply:
TEST 35 & 36 which actually checks the schema? Can we learn from this？https://github.com/apache/apisix/blob/6162ca0ef92445fdd15b7a213655887b66cd4eea/t/plugin/request-validation.t#L27-L45

Reply:
Can the following check schema be implemented in a similar way?

Reply:
Sure